### ISSUE TITLE
Integrate with Key?

### ISSUE THREAD

**Issue by laryn** [backdrop-contrib/stripe#28]
I haven't looked into this yet but wanted to make a note that this module should perhaps be integrated with [Key](https://github.com/backdrop-contrib/key) for the API keys.

**Comment by laryn** [comment]
I just posted this in a thread in Zulip and thought I'd paste it here as well.

> ...long term I'd like to move sensitive config like this to support the Key module so the details can be more easily saved elsewhere (e.g. in a file that is not tracked in version control). That said, if you use `settings.local.php` you should be able to define that value there as a config override already now, in a way that avoids storing it in regular config that might be in your repo.

**Comment by jenlampton** [comment]
I'd love to see (optional) integration with Key.

**Comment by stpaultim** [comment]
Yes, we'd like to see this as well. Let us know how we can help?

**Comment by abrink33** [comment]
Having the option to integrate with Key would be great!
